e2e/test-component/scenarios/embedding-sdk/styles-tests.cy.spec.tsx

```diff
@@ -1,18 +1,20 @@
 import {
+  CreateDashboardModal,
+  InteractiveQuestion,
   MetabaseProvider,
   StaticQuestion,
 } from "@metabase/embedding-sdk-react";
 
 import { ORDERS_QUESTION_ID } from "e2e/support/cypress_sample_instance_data";
-import { describeEE, updateSetting } from "e2e/support/helpers";
+import { describeEE, modal, updateSetting } from "e2e/support/helpers";
 import {
   DEFAULT_SDK_PROVIDER_CONFIG,
   mockAuthProviderAndJwtSignIn,
   signInAsAdminAndEnableEmbeddingSdk,
 } from "e2e/support/helpers/component-testing-sdk";
 import { getSdkRoot } from "e2e/support/helpers/e2e-embedding-sdk-helpers";
 
-describeEE("scenarios > embedding-sdk > static-dashboard", () => {
+describeEE("scenarios > embedding-sdk > styles", () => {
```

Comment:
this was probably a copy paste mistake, this file is not related to static dashboards